smart-doc/README.md

```diff
@@ -0,0 +1,176 @@
+tasks/systemd-timer-troubleshooting.xml
+concepts/network-ipv6-vs-ipv4.xml
+glues/systemd-timer-intro.xml
+glues/systemd-timer-whats-next.xml
+glues/systemd-timer-more-info.xml
+```
+
+The second component of the article should be an abbreviated title. As our articles are intended to address specific problems and tasks, use verbs in gerund here as well:
+
+```
+articles/systemd-timer-working-with.asm.xml
+articles/autoyast-understanding-config-file.asm.xml
+```
+
+Distinguish article files from standard XML files by adding the `.asm` suffix.
+
+
+### ID naming conventions
+
+Working with assemblies, you need to distinguish between two types of IDs:
+
+* `external IDs`: These are visible to the world.
+* `internal IDs`: Internal references inside the assembly used to organize the content snippets.
+
+External IDs are built in a similar fashion as file names. As with file names, never include the 
+topic type in your ID.
+
+```
+<NAME> := <TOPIC>[-<SUBTOPIC]-<SNIPPET-TOPIC>-<TITLE>
+```
+
+To build an internal ID, prefix the external ID with an underscore (`_`).
+
+
+### Image naming conventions
+
+Make sure your images can be associated with the topic they belong to. Use a similar naming scheme 
+for images that you use for topic files.
+
+
+```
+<NAME> := <TOPIC>[-<SUBTOPIC]-<SNIPPET-TOPIC>-<TITLE>
+```
+## Version info
+
+We maintain a changelog for each article. Add a version to the changelog whenever there is a 
+*release* of your article, i.e. whenever you ask for a rebuild and repost of said article. Do not 
+artificially bloat the changelog by entering every single commit. Also, provide concise and 
+meaningful version information that the reader benefits from. This information might one day end up 
+being sent out to readers as update notifications on content they subscribed to.
+
+## Metadata
+
+Once your article is done, add a metadata layer to the assembly file. Check the `assembly.asm.xml` 
+file for XML synthax and possible values. The following metadata types are currently supported:
+
+* Internal
+* SEO
+* Search
+* Publishing info
+* Social media
+
+### Internal 
+
+1. Add maintainer info (e-mail address) to the assembly and all the topic files. Use this information to contact the original author, in case you want to reuse a piece and would like to start a conversation on changing the file.
+1. Determine whether the article needs to be translated and add a list of languages.
+1. If you have reused the content from existing legacy (non-modular) docs, adjust the pointer in the 
+XML comments of your file. By adding a similar pointer to the legacy doc piece, you make sure that 
+you keep both docs in sync. This is done using a plain XML comment.
+
+### SEO
+
+Add some search engine-related information to your file:
+1. `title`- 29-55 characters
+1. `description` - max. 150 characters
+
+### Search
+
+Add as much search-related data to your article as possible. Choose the appropriate tags:
+* `productname`
```

Comment:
Same here:

```suggestion
Add as much search-related data to your article as possible. Choose the appropriate tags:

* `productname`
```

Reply:
Integrated in https://github.com/openSUSE/doc-kit/pull/78.